sdks/python/hatchet_sdk/connection.py

```diff
@@ -60,7 +60,7 @@ def new_conn(config: ClientConfig, aio: bool) -> grpc.Channel | grpc.aio.Channel
 
     # Set environment variable to disable fork support. Reference: https://github.com/grpc/grpc/issues/28557
     # When steps execute via os.fork, we see `TSI_DATA_CORRUPTED` errors.
-    os.environ["GRPC_ENABLE_FORK_SUPPORT"] = "False"
+    os.environ["GRPC_ENABLE_FORK_SUPPORT"] = config.grpc_enable_fork_support
```

Comment:
If a user enables fork support, do we want to set `GRPC_POLL_STRATEGY=poll` or `GRPC_POLL_STRATEGY=epoll1`?

From reference thread: https://github.com/grpc/grpc/issues/28557#issuecomment-1022710189